app/views/user/contribute/_completed_tracks.erb

```diff
@@ -0,0 +1,19 @@
+<h4>You have completed the current tracks, you can contribute by adding a new exercise from another language track, below you will find the link for each finished track</h4>
```

Comment:
I think this will be easier to understand if we have a list rather than a table. Imagine if we have 5 languages, it would look like this:

|  Completed Tracks | Link  | 
|---|---|
| ruby  | [full list of unimplemented exercises in ruby](http://exercism.io/languages/ruby/contribute) |
| csharp | [full list of unimplemented exercises in csharp](http://exercism.io/languages/csharp/contribute)  |
| python | [full list of unimplemented exercises in python](http://exercism.io/languages/python/contribute) |
| objective-c | [full list of unimplemented exercises in objective-c](http://exercism.io/languages/objective-c/contribute) |
| cpp | [full list of unimplemented exercises in cpp](http://exercism.io/languages/cpp/contribute) |

How about something like this?

```
You've completed the following tracks. Want to help add more exercises?

* [Language Name](link to the list of unimplemented problems)
```